read_zdf_vis_3d.py

```diff
@@ -42,14 +36,76 @@ def _main():
     )
     plt.colorbar()
     plt.title("Depth map")
-    plt.show()
+    plt.show(block=False)
+
+
+def _get_mid_point(xyz):
+    """Calculate mid point from average of the 100 centermost points.
+
+    Args:
+        xyz: X, Y and Z images (point cloud co-ordinates)
+
+    Returns:
+        mid_point: Calculated mid point
+
+    """
+    xyz_center_cube = xyz[
+        int(xyz.shape[0] / 2 - 5) : int(xyz.shape[0] / 2 + 5),
+        int(xyz.shape[1] / 2 - 5) : int(xyz.shape[1] / 2 + 5),
+        :,
+    ]
+    return (
+        np.nanmedian(xyz_center_cube[:, :, 0]),
+        np.nanmedian(xyz_center_cube[:, :, 1]),
+        np.nanmedian(xyz_center_cube[:, :, 2]),
+    )
+
+
+def _display_pointcloud(rgb, xyz):
+    """Display point cloud.
+
+    Display the provided point cloud `xyz`, and color it with `rgb`.
+
+    We take the centermost co-ordinate as 'lookat' point. We assume that
```

Comment:
That would be good. However, I wanted to use it as a variable name, and then it can't be with a dash. In the comments I refer to the variable with the markdown notation `lookat`. Maybe I can use `look_at` instead? Or I can disregard me wanting to use the variable name in documentation.